packages/http/src/mocker/__tests__/HttpMocker.spec.ts

```diff
@@ -229,6 +228,118 @@ describe('HttpMocker', () => {
             expect(response.body).toEqual((selectedExample as INodeExample).value);
           });
         });
+
+        describe('and a response example is defined, but no response example is requested', () => {
+          it('returns the first example', () => {
+            const response = httpMocker.mock({
+              input: mockInput,
+              resource: mockResource,
+              config: { mock: { dynamic: false } },
+            });
+
+            expect(response.body).toBeDefined();
+
+            const selectedExample = flatMap(mockResource.responses, res =>
```

Comment:
Mind simplifying it to

```suggestion
            const selectedExample = mockResource.responses[0].contents[0].examples[0]
```

?

Using flat map is cool but not correct if we had other responses defined in that object (400, 500).

Reply:
Absolutely, I think this is a result of a copy paste that I adapted.